packages/expo-image/src/Image.types.ts

```diff
@@ -308,51 +292,44 @@ export interface ImageNativeProps extends ImageProps {
  */
 export type ImageContentPositionValue = number | string | `${number}%` | `${number}` | 'center';
 
-// eslint-disable
-// prettier-ignore
 /**
- * Specifies the position of the image inside its container. One value controls the x-axis and the second value controls the y-axis.
+ * Specifies the position of the image inside its container. One value controls the X axis and the second value controls the Y axis.
+ *
+ * Additionally, it supports stringified shorthand form that specifies the edges to which to align the image content,
+ * which possible values are represented by [`ImageContentPositionString`](#imagecontentpositionstring) string union.
```

Comment:
```suggestion
 * which possible values are represented by [`ImageContentPosition`](#imagecontentposition) string union.
```

I think "String" part wasn't intentional here. The link to `ImageContentPosition` type is: https://docs.expo.dev/versions/latest/sdk/image/#imagecontentposition

Reply:
This is an intended change, I have unhidden this type in the Image code:
* https://github.com/expo/expo/pull/22289/files#diff-2d89ae8261a9bd9e2812270ef5f4198e36ee0a6bcd60521b2acbbcefc54aa4c3L355-L357

Reply:
Another place that I think we agreed that separating these types is unnecessary and makes the docs less clear and require the user to unnecessarily jump between types

Reply:
Those types are listed directly below each other, and the description of `ImageContentPosition` already seems for me a bit lengthy, but since you have a strong opinion on that, I'm going to revert the comments to the previous state.